packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/AnnotationResource.java

```diff
@@ -0,0 +1,59 @@
+package software.uncharted.terarium.hmiserver.resources;
+
+import io.quarkus.security.Authenticated;
+import io.quarkus.security.identity.SecurityIdentity;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import software.uncharted.terarium.hmiserver.entities.Annotation;
+
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/api/annotations")
+@Authenticated
+@Tag(name = "User Annotation REST Endpoints")
+@Slf4j
+public class AnnotationResource {
+	@Inject
+	SecurityIdentity securityIdentity;
+
+	@GET
+	public Response getAnnotations(
+			@QueryParam("artifact_type") final String artifactType,
+			@QueryParam("artifact_id") final String artifactId) {
+
+			System.out.println("");
```

Comment:
Probably want to remove debug statements here, or at least, replace with `log` if this is wanted output.